cypress/integration/challenge-tests/blocks/apis-and-microservices-projects.js

```diff
@@ -0,0 +1,18 @@
+/* global cy */
+const superblockPathData = require('../../../fixtures/pathData/apis-and-microservices.json');
+
+const challengePaths =
+  superblockPathData['blocks']['apis-and-microservices-projects'];
+
+challengePaths.forEach(challenge => {
+  let challengeName = challenge.split('/');
+
+  it(
+    'Challenge ' +
+      challengeName[challengeName.length - 1] +
+      ' should work correctly',
+    () => {
+      cy.testChallenges(challenge);
```

Comment:
So, my guess is that we'll need to do the `visit` in a before hook, since doing that seems to have eliminated the flakiness from `main`.

I don't understand *why* `visit`ing in `it` causes problems, but it seems to fairly consistently.

Reply:
> Flakiness aside, this is looking good.
> 
> Btw, we shouldn't need to commit the block specs if we're generating them dynamically. We can .gitignore them.

I think we should commit them if people want to write tests for specific blocks.

Reply:
> So, my guess is that we'll need to do the `visit` in a before hook, since doing that seems to have eliminated the flakiness from `main`.
> 
> I don't understand _why_ `visit`ing in `it` causes problems, but it seems to fairly consistently.

So I can't reproduce these errors locally but it's worth a try.